src/.vuepress/components/AppDownload.vue

```diff
@@ -12,7 +12,21 @@
 <script>
 export default {
   name: "appdownload",
-  props: {}
+  props: {},
+  methods: {
+    gtmEvent: function (store) {
+      if (!this.$gtm) return
+
+      this.$gtm.trackEvent({
+        event: null, // Event type [default = 'interaction'] (Optional)
```

Comment:
Are these resulting values all in-line with the conventions we use at present in GA and friends?

Reply:
It matches GTM docs and I tried to follow conventions from other websites (label, action, category).

But value does not make sense to me, I will remove it.